check_stability.py

```diff
@@ -172,6 +172,9 @@ def version(self):
     def wptrunner_args(self):
         return NotImplemented
 
+    def extra_setup(self):
+        pass
```

Comment:
The term "setup" is a bit ambiguous in this context (i.e. more installation
steps?) Could we rename the `extra_setup` method to `prepare_environment`? And
defer its invocation to just prior to `run_tests`?

Also, could you add a doc string?